frontera/contrib/scrapy/converters.py

```diff
@@ -26,7 +28,7 @@ def to_frontier(self, scrapy_request):
         if callable(eb):
             eb = _find_method(self.spider, eb)
 
-        scrapy_meta = scrapy_request.meta
+        scrapy_meta = dict_to_bytes(scrapy_request.meta, keys_only=True)
```

Comment:
Although this won't make a difference to frontera since it currently doesn't access scrapy_meta, but we might want to keep the conversion of keys(and also values) to bytes to follow our convention of having bytes in meta.